packages/st2/Makefile

```diff
@@ -117,8 +142,7 @@ endif
 wheelhouse: .build-runners .stamp-wheelhouse
 .stamp-wheelhouse: | populate_version requirements inject-deps
 	cat requirements.txt
-	pip wheel --wheel-dir=$(WHEELDIR) --find-links=$(WHEELDIR) -r requirements.txt || \
-		pip wheel --wheel-dir=$(WHEELDIR) --find-links=$(WHEELDIR) -r requirements.txt
```

Comment:
Please keep the original version with `||`, it's there to help retrying the build if first `pip wheel` fails.

Reply:
ok - that wasn't obvious. Adding a comment about that also